src/repositories/metadata-repository.ts

```diff
@@ -47,6 +47,7 @@ export class MetadataRepository implements IMetadataRepository {
    * Store metadata entry to the database.
    */
   async save(entry: MetadataInput): Promise<void> {
+    console.log('save', entry, MetadataInput.encode(entry))
```

Comment:
Should this be a logger call, or should it be removed?

Reply:
Dang. Yes. Thank you for noticing this.